consensus/state_sync/module.go

```diff
@@ -37,6 +38,8 @@ type StateSyncModule interface {
 	IsServerModEnabled() bool
 	EnableServerMode()
 	SetLogPrefix(string)
+
+	SendStateSyncMessage(*typesCons.StateSyncMessage, cryptoPocket.Address, uint64) error
```

Comment:
Is this only exposed for debug reasons?

Reply:
no, it is used by state sync functions as well to send Metadata and Block requests and responses.